force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -161,18 +223,24 @@ public with sharing class FlowControlLoopsCollections {
      * @return The new string with the inputString repeated repeatCount times.
      */
     public static String repeatString(String inputString, Integer repeatCount) {
-        // Initialize the result String
+       String finString = ''; // Initialize the result String
 
         // Use a for loop to append the inputString to the result string repeatCount times
-        /*
-        for () {
-
-            // If it is not the last iteration, add a semicolon to separate the strings
+        
+        for (Integer i = 0; i < repeatCount; i++) {
+            if(i == repeatCount - 1){ //No semicolon on the last iteration
+                finString = finString + inputString;
+                System.debug(i);
+                System.debug(finString);
+            }
+            else {  // If it is not the last iteration, add a semicolon to separate the strings
+                finString = finString + inputString + '; ';
+                System.debug(i);
+                System.debug(finString);
```

Comment:
Generally you should remove System.debug() and console logging before you issue a PR for your work. In some cases and some companies, it doesn't matter, but especially for frontend work this is considered not a great practice. For backend, it's less of an issue, and depending on your company they may not care.